srcCpp03/RTIDDSImpl.cxx

```diff
@@ -1238,7 +1534,7 @@ dds::core::QosProvider RTIDDSImpl<T>::getQosProviderForProfile(
 
     if (!_PM->get<bool>("noXmlQos")) {
         qosProvider = dds::core::QosProvider(
-                _PM->get<std::string>("qosFile").c_str(),
+                _PM->get<std::string>("qosFile"),
```

Comment:
Was this working before? Was this a bug? If so, I want to report it and add in the documentation (Release Notes) that it is fixed.

Reply:
Yes, it was working because `dds::core::QosProvider` gets two `std::string` as parameters.
You can assign a `const char *` to a string. It will convert it to an actual string.